src/applications/facility-locator/utils/helpers.jsx

```diff
@@ -304,3 +304,46 @@ export const recordResultEvents = (location, index) => {
     recordEvent({ 'fl-facility-id': id });
   }
 };
+
+/**
+ * Helper fn to record search data layer
+ */
+export const recordSearchEvents = (data, meta) => {
+  if (data && data.length > 0) {
+    recordEvent({ event: 'fl-search-results' });
+    recordEvent({ 'fl-results-returned': true });
+  } else if (data.length === 0) {
+    recordEvent({ 'fl-results-returned': false });
```

Comment:
I think you could get an error here if `data` is null or undefined.

How about just:

```js
if (data) {
    recordEvent({ event: 'fl-search-results' });
    recordEvent({ 'fl-results-returned': data.length })
}
```